benefits/enrollment/views.py

```diff
@@ -21,6 +21,7 @@
 ROUTE_INDEX = "enrollment:index"
 ROUTE_REENROLLMENT_ERROR = "enrollment:reenrollment-error"
 ROUTE_RETRY = "enrollment:retry"
+ROUTE_SERVER_ERROR = "core:server-error"
```

Comment:
Does it feel weird to have this definition here? Should it be imported from `core.urls` or something?